### PR TITLE
SDANSA-95 Create CV for publishing destinations

### PR DESCRIPTION
**Pull request by mugurrus**

### CODE REVIEW

File: superdesk/metadata/item.py [hunk 1]
```diff
@@ -557,6 +557,9 @@
     'assignment_id': {
         'type': 'string',
         'mapping': not_analyzed
+    },
+    'destination': {
```
Comment (petrjasek) [inline superdesk/metadata/item.py]: why is this needed?

Comment (mugurrus) [reply superdesk/metadata/item.py]: To store the publishing destination

Comment (petrjasek) [reply superdesk/metadata/item.py]: but why not use custom vocabularies?
